### DISCUSSION TITLE
Create simple apps that showcase Remix PWA APIs (ideas open)

### DISCUSSION THREAD

**Discussion by ShafSpecs** [ShafSpecs/remix-pwa#113] [Ideas]
Currently, the `README` is the only place to get information on remix-pwa APIs. Customizing your PWA further is difficult due to lack of examples, and some APIs are not really clear/implemented well (e.g Push API)

**Ideas:** 
- Create separate examples that showcases multiple features like push api, background sync, caching strategies, etc.
- Create a standalone documentation for remix-pwa

Currently open to more ideas and suggestions to improve the overall DX

**Comment by ShafSpecs** [comment]
Currently, a standalone documentation is currently in the works which should showcase remix-pwa and future packages that would come along.
https://github.com/remix-pwa/remix-pwa/issues/46

**Comment by JamesBrandts** [comment]
This is a great idea, if I could only see the app from the push tutorial working, then then I could start changing things and see what breaks it and what doesn't. I tried creating a new remix app and following the sw, then the push tutorial without success, if I can start from the app working, it's easier to see what broke it, when something goes wrong.
